i3_agenda/__init__.py

```diff
@@ -150,12 +150,16 @@ def getEvents(service, allowed_calendars_ids: List[str], max_results: int, today
             end_time = get_event_time(event['end'].get('dateTime', event['end'].get('date')))
             start_time = event['start'].get('dateTime', event['start'].get('date'))
             unix_time = get_event_time(start_time)
+            try:
```

Comment:
instead of try catch, a better approach would be
``` python
location = None
if 'location' in event:
    location = event['location']
```

Reply:
> instead of try catch, a better approach would be
> 
> ```python
> location = None
> if 'location' in event:
>     location = event['location']
> ```